### ISSUE TITLE
Adafruit Feather RP2040 RFM95 Initialization Issue

### ISSUE THREAD

**Issue by loganfarrow** [jgromes/RadioLib#871]
**Sketch that is causing the module fail**

```c++
#include <RadioLib.h>

// Define the pin mapping for the RFM95 LoRa module
#define RFM95_CS    16  // Chip select pin
#define RFM95_RST   17  // Reset pin
#define RFM95_IRQ   21  // Interrupt pin, connected to DIO0
#define RFM95_GPIO  22  // Additional GPIO, connected to DIO1 

SX1272 radio = new Module(RFM95_CS, RFM95_IRQ, RFM95_RST, RFM95_GPIO);

void setup() {

  pinMode(LED_BUILTIN, OUTPUT);
  pinMode(RFM95_RST, OUTPUT);
  digitalWrite(RFM95_RST, HIGH);

  Serial.begin(115200);
  while (!Serial) delay(1);
  delay(100);

  Serial.println("Feather LoRa RX Test!");

  // manual reset
  digitalWrite(RFM95_RST, LOW);
  delay(10);
  digitalWrite(RFM95_RST, HIGH);
  delay(10);





  int state = radio.begin();
  if (state == RADIOLIB_ERR_NONE) {
    // no error
    Serial.println(F("Initialization successful!"));
  } else {
    // some error occurred
    Serial.print(F("Initialization failed, code "));
    Serial.println(state);
    while (true);
  }



}

void loop() {
  Serial.print("Sending message...");
  // Send a string "Hello, World!".
  int state = radio.transmit("Hello, World!");
  if (state == RADIOLIB_ERR_NONE) {
    Serial.println("success!");
  } else {
    Serial.print("failed, error code: ");
    Serial.println(state);
  }

  // Wait for a bit before sending the next message.
  delay(1000);
}
}
```

**Hardware setup**
[Wiring diagram, schematic, pictures etc.](https://github.com/adafruit/Adafruit-Feather-RP2040-RFM-PCB/blob/main/Adafruit%20Feather%20RP2040%20RFM9x%20Pinout.pdf)

**Debug mode output**

RadioLib Debug Info
Version:  6.2.0.0
Platform: Raspberry Pi Pico (unofficial)
Compiled: Nov  6 2023 17:28:08

R	42	0	
SX127x not found! (1 of 10 tries) RADIOLIB_SX127X_REG_VERSION == 0x0000, expected 0x0022
R	42	0	
SX127x not found! (2 of 10 tries) RADIOLIB_SX127X_REG_VERSION == 0x0000, expected 0x0022
R	42	0	
SX127x not found! (3 of 10 tries) RADIOLIB_SX127X_REG_VERSION == 0x0000, expected 0x0022
R	42	0	
SX127x not found! (4 of 10 tries) RADIOLIB_SX127X_REG_VERSION == 0x0000, expected 0x0022
R	42	0	
SX127x not found! (5 of 10 tries) RADIOLIB_SX127X_REG_VERSION == 0x0000, expected 0x0022
R	42	0	
SX127x not found! (6 of 10 tries) RADIOLIB_SX127X_REG_VERSION == 0x0000, expected 0x0022
R	42	0	
SX127x not found! (7 of 10 tries) RADIOLIB_SX127X_REG_VERSION == 0x0000, expected 0x0022

**Additional info (please complete):**
 - MCU: RP2040
 - Link to Arduino core: [e.g. https://github.com/stm32duino/Arduino_Core_STM32 when using official STM32 core. See readme for links to all supported cores]
 - Wireless module type SX1272?
 - Arduino IDE version  2.2.1
 - Library version most recent
 
 
 Hi there, The issue I am encountering is that I can initialize my module only as SX1276 or SX1278 and not SX1272 (even though all the Adafruit information tells me this is an RFM95 915Mhz with SX1272) I get the error -2 saying that I am trying to use the wrong chip. Transmission between my feathers works correctly when using them as an SX1278 on 434Mhz, but I then I am not able to set the freq to use 915Mhz. 
 
 TLDR: Adafruit tells me this is a RFM95 915Mhz (printed on PCB) but RadioLib does not recognize it as SX1272. 
 
 See information and board I am working with here: https://learn.adafruit.com/feather-rp2040-rfm95/downloads

**Comment by loganfarrow** [comment]
For anyone struggling like I was:

Adafruit documentation is incorrect on the RFM95's included SX1272. Initialize it in RadioLib as a SX1276 to access 915Mhz.
